x-pack/plugins/lens/public/datasources/form_based/dimension_panel/dimension_panel.test.tsx

```diff
@@ -154,6 +157,18 @@ const bytesColumn: GenericIndexPatternColumn = {
   params: { format: { id: 'bytes' } },
 };
 
+const services = coreMock.createStart() as unknown as LensAppServices;
```

Comment:
Marco says this would be a code smell in prod code, but this double type cast is ok in a test.